joinmarket/configure.py

```diff
@@ -116,11 +116,16 @@ def jm_single():
 socks5_host = localhost
 socks5_port = 9050
 #for tor
-#host = 6dvj6v5imhny3anf.onion
+#newnym = true
+#newnym_delay = 60
+#tor_host = localhost
+#tor_port = 9051
```

Comment:
A better name for these may be `tor_control_host`, at first I thought it was another name for socks5 port.